lib/Support/Unix/Process.inc

```diff
@@ -95,7 +95,10 @@ unsigned Process::getPageSize() {
 size_t Process::GetMallocUsage() {
 #if defined(HAVE_MALLINFO)
   struct mallinfo mi;
+#pragma clang diagnostic push
```

Comment:
nit: Guessing this unrelated piece should be rebased onto your other PR?

Reply:
Removed.